utils/fetchData.js

```diff
@@ -0,0 +1,21 @@
+export const fetchData = async ({URL, query}) => {
+    try
+    {
+        const attributes = Object.keys(query);
+        if(attributes.length > 1)
+        {
+            URL += `?`+ attributes[0] + `=${query[attributes[0]]}&` + attributes[1] + `=${query[attributes[1]]}`;
+        }
+        else if(attributes.length > 0)
+        {
+            URL += `?`+ attributes[0] + `=${query[attributes[0]]}`;
+        }
```

Comment:
improve this condition. What if i have a third query param?

Reply:
let postfix = ""
loop the attributes
If postfix length is 0, add a "?" else "&"
Add the attribute and its value